crates/primitives/src/receipt.rs

```diff
@@ -56,6 +56,12 @@ impl Receipt {
     pub fn with_bloom(self) -> ReceiptWithBloom {
         self.into()
     }
+
+    /// Calculates the bloom filter for the receipt and returns the [ReceiptWithBloomRef] container
+    /// type.
+    pub fn with_bloom_ref(&self) -> ReceiptWithBloomRef<'_> {
```

Comment:
as_bloom_ref ? maybe

Reply:
thats what the type is called, it would also make more sense to highlight that this is slow and not have Into impls